basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/ServletJSPSettingsTestCase.java

```diff
@@ -33,7 +30,7 @@
     private final String JAVA_ENCODING = "java-encoding";
     private final String MAPPED_FILE = "mapped-file";
     private final String SCRATCH_DIR = "scratch-dir";
-    private final String SMAP = "_smap";
+    private final String SMAP = "smap";
```

Comment:
Is it really working with removed opening underscore? IMO it is there to distinguish between ids like `form-gwt-uid-585_smap` and `form-gwt-uid-585_dump-smap`. If the underscore is still necessary and you will keep it, please add related comment why this is here.